### PR TITLE
Prepare readme for 2019

### PR DESCRIPTION
**Pull request by soc1c**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
-* * *
+- Find a team on Github: https://github.com/ethberlinzwei/Find-A-Team
+- Connect with hackers on Riot: [#ethberlinhackers:matrix.org](https://webchat.weho.st/#/room/#ethberlinhackers:matrix.org)
+- Contribute to the knowledge base on Riot: [#ethberlinzwei-kb:chat.weho.st](https://webchat.weho.st/#/room/#ethberlinzwei-kb:chat.weho.st)
+- Sign up for ETHBerlin Zwei on Devpost: https://ethberlin.devpost.com/
+- Check for updates on: https://ethberlinzwei.com/
+- Find the old knowledge base from last year at: https://github.com/ethberlin-hackathon/ETHBerlin-KnowledgeBase
 
-# ETHBerlin Knowledge Base
+### @TODO for this knowlege base 2019
```
Comment (ligi) [inline README.md]: typo knowlege - knowledge

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — APPROVED** [review]

**Comment by ChrisChinchilla** [comment]
OK, this will give me some merge conflicts with all the stuff I had waiting to push and the todos were there for a reason… But thanks!

**Comment by ligi** [comment]
ah - oh dam - let's coordinate it in the riot room better then and avoid such things in the future. Perhaps we do locking on pages and separate work by pages?

**Comment by ChrisChinchilla** [comment]
Ahh, you moved the TODOs, OK. Actually, I have a bot that can do this automatically, should add it.

**Comment by ChrisChinchilla** [comment]
@ligi I just take a bit longer to go through emails and notifications, it's OK ;)

**Comment by ligi** [comment]
great - let's just not waste time with merges - life is to precious to waste it with conflicts ;-)

**Comment by soc1c** [comment]
please open PRs for thing you are working on. otherwise it's difficult to see who's working on what.

**Comment by ChrisChinchilla** [comment]
@soc1c Sure, I wasn't necessarily expecting any other contributions, and this was stuff that wasn't even started and thus no PRs yet. But anyway, this all merged cleanly so there's no problem really.

**Pull request merged**